-------------------------------------------------------------------------------
Test set: controller.TesteFuncionarios
-------------------------------------------------------------------------------
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 1.159 s <<< FAILURE! - in controller.TesteFuncionarios
controller.TesteFuncionarios  Time elapsed: 1.156 s  <<< ERROR!
java.lang.RuntimeException: java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
	at controller.TesteFuncionarios.setUpClass(TesteFuncionarios.java:27)
Caused by: java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
	at controller.TesteFuncionarios.setUpClass(TesteFuncionarios.java:27)

